plugins/dynamodb/komand_dynamodb/util/utils.py

```diff
@@ -0,0 +1,202 @@
+import base64
+import datetime
+import json
+
+import botocore.response
+
+from insightconnect_plugin_runtime.exceptions import PluginException
+
+
+class Utils:
+    @staticmethod
+    def prepare_input_for_aws_call(params: dict, schema: dict) -> dict:
+        try:
+            params = Utils.format_input(params, schema)
+        except Exception:
+            raise PluginException(
+                cause="Unable to format input parameters.",
+                assistance="Please check the inputs and try again.",
+            )
+        return params
+
+    @staticmethod
+    def format_input(params: dict, schema: dict):
+        formatted_params = {}
+        # Drop invalid empty parameters
+        for k, v in params.items():
+            if v:
+                formatted_params[k.replace("$", "")] = v
+        return Utils.convert_all_dict_keys_to_upper_camel_case(formatted_params, schema, schema)
+
+    @staticmethod
+    def convert_all_dict_keys_to_upper_camel_case(
+        dict_to_convert: any, root_schema: dict, schema: dict, has_lower_level=False
```

Comment:
I have tried to make it more precise, however. Sometimes there can be a dict and there can be a str. This dict is input from user. I have thought about how to do this as precisely as possible, but there is a feeling that it cannot be specified. Alternatively, I can simply delete any to avoid confusion